src/Serialization/containers.jl

```diff
@@ -29,13 +40,23 @@ end
 
 # deserialize without specific content type
 function load_internal(s::DeserializerState, ::Type{Vector}, dict::Dict)
+    if haskey(dict, :entry_type)
+        T = decodeType(dict[:entry_type])
+        
+        return [load_type_dispatch(s, T, x) for x in dict[:vector]]
+    end
     return [load_unknown_type(s, x) for x in dict[:vector]]
 end
 
 function load_internal_with_parent(s::DeserializerState,
                                    ::Type{Vector},
                                    dict::Dict,
                                    parent)
+    if haskey(dict, :entry_type)
+        T = decodeType(dict[:entry_type])
+        
+        return [load_type_dispatch(s, T, x) for x in dict[:vector]]
```

Comment:
Is the parent intentionally ignored here? May make sense, if we assume "basic types" must have either no parent, or a unique parent -- but if we assume that, then we should (a) add a comment here stating that the `parent` is intentionally being ignored, and (b) add this information to the `is_basic_serialization_type` documentation

Reply:
this was actually a typo, thanks!